airootfs/usr/share/vx-img/flash-image.sh

```diff
@@ -12,6 +12,7 @@ function menu() {
     unset 'items[-1]'
 
     if [ ${#items[@]} -eq 0 ]; then
+        echo "Got an empty list!"
```

Comment:
maybe say more in this error message?

Reply:
This sets `err`, which is then accompanied with a more contextual error. This message was actually for debugging purposes, I could take it out...

Reply:
ah gotcha, yeah take it out that's cool.